weasyl/test/test_login.py

```diff
@@ -88,9 +89,10 @@ def testAuthenticateBcrypt_LoginFailsForModsWithInvalidAuthentication(tmpdir):
     log_path = '%s%s.%s.log' % (macro.MACRO_SYS_LOG_PATH, 'login.fail', d.get_timestamp())
     mod_userid = 2061
     user_id = db_utils.create_user(username='ikani')
-    d.execute("UPDATE login SET userid = %i WHERE userid = %i", [mod_userid, user_id])
+    d.engine.execute("UPDATE login SET userid = %(newid)s WHERE userid = %(oldid)s", newid=mod_userid, oldid=user_id)
```

Comment:
This feels a little off. Maybe modify `create_user` to accept a `userid` argument?